README.md

```diff
@@ -145,3 +145,17 @@ use ApplicationInsights::Rack::TrackRequest, '<YOUR INSTRUMENTATION KEY GOES HER
 # For rails, suggest to set up this middleware in application.rb so that unhandled exceptions from controllers are also collected
 config.middleware.use 'ApplicationInsights::Rack::TrackRequest', '<YOUR INSTRUMENTATION KEY GOES HERE>', <buffer size>
 ```
+
+#### Retrieving a request's AppInsight ID  ####
+```ruby
+# from time to time you may need to access a request's AppInsight id from within your app
+application_insights_request_id = env['ApplicationInsights.request.id']

```

Comment:
I don't know much about rack, but is environment variable enough? What if there are multiple request? The requestId should be saved on the request context in this case. But I may be wrong lack of knowledge about rack.

Reply:
The env value in this instance is unique to a given request so this should meet our requirements for storing the request id